src/components/Navmenu.tsx

```diff
@@ -3,11 +3,16 @@ import { Link } from '../renderer/Link'
 import { useTranslation } from 'react-i18next'
 import { Label } from './Label'
 import { cream, red, Paths } from '../constants'
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 
 export const Navmenu = () => {
-  const [url, setUrl] = useState<string>('/')
+  const [url, setUrl] = useState<string | undefined>(undefined)
```

Comment:
Why path initially is undefined?

Reply:
because I get if from the URL. If undefined it is not rendered

Reply:
fixed